src/js/views/patients/shared/components/owner_component.js

```diff
@@ -13,7 +13,7 @@ import './owner-component.scss';
 const i18n = intl.patients.shared.components.ownerComponent;
 
 const OwnerItemTemplate = hbs`<div>{{matchText name query}} <span class="owner-component__team">{{matchText short query}}</span></div>`;
-const FilterButtonTemplate = hbs`{{far "circle-user"}}<span>{{ name }}</span>{{far "angle-down"}}`;
+const FilterButtonTemplate = hbs`{{far "circle-user"}}<span class="owner-component__name">{{ name }}</span>{{far "angle-down"}}`;
```

Comment:
Added a class name here so I can target it for the ellipsis overflow CSS styling.

Reply:
Can we add a new flag for the owner component similar to `isCompact` and `isFilter` for this use of the component?  This way we can specifically set this overflow and remove the icon with a separate template and we don't have to nest `button-filter` and override the `button-filter` styles.